.env.development

```diff
@@ -1,14 +1,17 @@
 # NOTE: Variables need to be prefixed using REACT_APP_:
 # https://create-react-app.dev/docs/adding-custom-environment-variables/
 
-# REACT_APP_GATEWAY=http://localhost:8080/api/v1/
-REACT_APP_GATEWAY=https://acc.looplijst.top.amsterdam.nl/api/v1/
+# Commented variables are only to be used as examples
+# Add any local changes to `.env.development.local` so they won't be committed
+
+# Local API
+REACT_APP_GATEWAY=http://localhost:8080/api/v1/
```

Comment:
I changed it because we cannot use "permits" locally.
Decos doesn't allow to connect from a local machine.

But I realize Remco is not able to login yet.